content/updates/2018-jan.md

```diff
@@ -0,0 +1,21 @@
+---
+title: January 2018
+date: 2018-02-07
+hidden: true
+image: image  # pending pi-image
+update: update  # pending pi-image
+---
+
+- Support for [custom Arduino commands](/api/servo-assembly/arduino-commands/)
+- Implement [Cartesian coordinate system](/api/vision/coordinates/#cartesian-coordinates)
+- Fix issue with camera after code terminates
+- Remove dates from logs (They were wrong anyway)
+- Remove unnecessary noisy log messages
+- Remove unnecessary `size` attribute from a marker
+- Deprecate broken `polar` coordinate system in favour of [`spherical`](/api/vision/coordinates/#spherical-coordinates)
```

Comment:
Minor: Unfortunately the link under the code highlighting of `spherical` is non-obvious. Not sure we can do anything to fix that here though.